tests/integration/json_test.go

```diff
@@ -74,6 +74,16 @@ var _ = Describe("odojsonoutput", func() {
 			desiredDesCompJSON := `{"kind":"Component","apiVersion":"odo.openshift.io/v1alpha1","metadata":{"name":"nodejs","creationTimestamp":null},"spec":{"type":"nodejs","app": "myapp","source":"https://github.com/openshift/nodejs-ex"},"status":{"state":"Pushed"}}`
 			Expect(desiredDesCompJSON).Should(MatchJSON(actualDesCompJSON))
 
+			// odo storage create -o json
```

Comment:
Remove this storage scenario as it is covered in the respective cmd_storage_test.go file (https://github.com/openshift/odo/blob/master/tests/integration/cmd_storage_test.go#L123)